extensions-core/lookups-cached-global/src/main/java/io/druid/server/lookup/namespace/cache/NamespaceExtractionCacheManager.java

```diff
@@ -100,24 +58,14 @@ public NamespaceExtractionCacheManager(
                 .build()
         )
     );
-    ExecutorServices.manageLifecycle(lifecycle, listeningScheduledExecutorService);
-    this.serviceEmitter = serviceEmitter;
-    this.namespaceFunctionFactoryMap = namespaceFunctionFactoryMap;
-    listeningScheduledExecutorService.scheduleAtFixedRate(
+    ExecutorServices.manageLifecycle(lifecycle, scheduledExecutorService);
+    scheduledExecutorService.scheduleAtFixedRate(
         new Runnable()
         {
-          long priorTasksStarted = 0L;
-
           @Override
           public void run()
           {
             try {
-              final long tasks = tasksStarted.get();
-              serviceEmitter.emit(
-                  ServiceMetricEvent.builder()
-                                    .build("namespace/deltaTasksStarted", tasks - priorTasksStarted)
-              );
-              priorTasksStarted = tasks;
               monitor(serviceEmitter);
```

Comment:
I suggest to implement this as a separate PR in #3667 series, because this one is already very big